appp.js

```diff
@@ -0,0 +1,141 @@
+import products from './common/api.js';
+
+
+// render these three items on the screen as radio buttons with the same name and different values
+const radio1 = document.getElementById('product1');
+const radio2 = document.getElementById('product2');
+const radio3 = document.getElementById('product3');
+
+const radio1Span = document.getElementById('product1span');
+const radio2Span = document.getElementById('product2span');
+const radio3Span = document.getElementById('product3span');
+
+const img1 = document.getElementById('img1');
+const img2 = document.getElementById('img2');
+const img3 = document.getElementById('img3');
+
+const desc1 = document.getElementById('desc1');
+const desc2 = document.getElementById('desc2');
+const desc3 = document.getElementById('desc3');
+
+const productsData = products.slice();
```

Comment:
Since you're not mutating the original `products` array you don't really need to slice it. I would only keep this if you plan on keeping track of the votes and shown in the `productsData` array like it looks like you were planning on doing before.